src/services/auth/plugins/mobile/mobile.test.ts

```diff
@@ -34,7 +35,9 @@ const memberRawRepository = AppDataSource.getRepository(Member);
 const mockCaptchaValidation = (action: RecaptchaActionType) => {
   (fetch as jest.MockedFunction<typeof fetch>).mockImplementation(async () => {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Can we remove this ignore ?

Reply:
Sure, I'm not even sure what this is for.